<script>
    // 関数宣言の場合
    console.log(`3 + 5 = ${sum(3, 5)}`);
    // > 3 + 5 = 8
    function sum(val1, val2) {
        return val1 + val2;
    }
    console.log(`10 + 7 = ${sum(10, 7)}`);
    // > 10 + 7 = 17

    // 関数式の場合
    // console.log(`3 + 5 = ${sum(3, 5)}`);
    // // エラーが発生！
    // // > Uncaught ReferenceError: Cannot access ®
    
    // // エラーが発生したため、これ以降は実行されない
    // const sum = function (val1, val2) {
    //     return val1 + val2;
    // }
    // console.log(`10 + 7 = ${sum(10, 7)}`);
</script>